From 16692b781dec07213e6fd0399a4a3c71f23e216f Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Tue, 12 Sep 2017 11:20:22 +0800
Subject: [PATCH 2544/5242] MLK-16474 net: fec: fixup: remove the unnecessary
 variable

commit  2bce357b2693ae4ae46fc0131e4a5ddc5adcafb7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove the unnecessary variable.

(BuildInfo: SCFW 3e70523d, IMX-MKIMAGE 0, ATF 0)

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Reviewed-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/freescale/fec_fixup.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fec_fixup.c b/drivers/net/ethernet/freescale/fec_fixup.c
index da9e03d..097de6f 100644
--- a/drivers/net/ethernet/freescale/fec_fixup.c
+++ b/drivers/net/ethernet/freescale/fec_fixup.c
@@ -56,7 +56,6 @@ static int ar8031_phy_fixup(struct phy_device *dev)
 void fec_enet_register_fixup(struct net_device *ndev)
 {
 	struct fec_enet_private *fep = netdev_priv(ndev);
-	static int registered = 0;
 	int err;
 
 	if (!IS_BUILTIN(CONFIG_PHYLIB))
@@ -71,7 +70,7 @@ void fec_enet_register_fixup(struct net_device *ndev)
 					ar8031_phy_fixup);
 		if (err)
 			netdev_info(ndev, "Cannot register PHY board fixup\n");
-		registered = 1;
+		ar8031_registered = 1;
 	}
 }
 
-- 
1.7.9.5

